src/content/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works.mdx

```diff
@@ -394,7 +394,7 @@ The headers also contain information that helps us link the spans together later
 
 This header information is passed along each span of a trace, unless the progress is stopped by something like middleware or agents that don't recognize the header format (see Figure 1).
 
-![Diagram of a failed trace with proprietary headers.](./images/doc-5207_middleware_0.png "middleware.png")
+![Diagram of a failed trace with proprietary headers.](./images/doc-5207_middleware_0.png "Diagram of a failed trace with proprietary headers.")
```

Comment:
This is your choice, but I would highly recommend you change the image filename from `doc-5207_middleware_0` to something more meaningful because there are 4 images that start with "doc-5207" in this section.